Makefile

```diff
@@ -0,0 +1,167 @@
+		echo "Old version number $(VERSION)  same as new version number $(VERSIONNEXT)"
+		echo "nothing to do, aborting."
+	else
+		# we need to split the $(VERSION) date format into substrings to using
+		# sed. This is due to / being a special character in sed.
+		YEAR=$(shell echo $(VERSIONDATE) | cut -d "/" -f 1)
+		MONTH=$(shell echo $(VERSIONDATE) | cut -d "/" -f 2)
+		DAY=$(shell echo $(VERSIONDATE) | cut -d "/" -f 3)
+		# update version info and date of *.sty, *.cls and *.tex files
+		# prepare find and replace with sed
+		findstr="$$YEAR\\/$$MONTH\\/$$DAY $(VERSION)"
+		replacestr="$(VERSIONDATENEXT) $(VERSIONNEXT)"
+		for currentdir in $(MODERNCVDIR) $(MANUALDIR); do
+			for file in $$currentdir/*.cls $$currentdir/*.sty $$currentdir/*.tex; do
+				if [[ -f "$$file" ]] && [[ ! -h "$$file" ]]; then
+					echo "updating version info of file $$file to $(VERSIONNEXT) (was $(VERSION))";
+					sed -i "s/$$findstr/$$replacestr/g" $$file;
+					# update version info in the title of documentation
+					sed -i "s/Package version $(VERSION)}/Package version $(VERSIONNEXT)}/g" $$file;
+				fi
+			done
+		done
+		unset findstr; unset replacestr
+		# update VERSION and VERSIONDATE variable of this very file
+		sed -i "s/VERSION = $(VERSION)/VERSION = $(VERSIONNEXT)/g" $(MODERNCVDIR)/Makefile
+		findstr="VERSIONDATE = $$YEAR\\/$$MONTH\\/$$DAY"
+		replacestr="VERSIONDATE = $(shell date +"%Y")\\/$(shell date +"%m")\\/$(shell date +"%d")"
+		sed -i "s/$$findstr/$$replacestr/g" $(MODERNCVDIR)/Makefile
+		unset findstr; unset replacestr
+	fi
+
+tarball:
+	# build a tarball for release puposes. If the examples directory exist,
+	# include them
+	# remove existing tarballs
+	rm -f *.gz *.tar
+	# create tar with all files in git repo
+	git archive HEAD > $(TARBALL)
+	# remove git specific files
+	tar -f $(TARBALL) --delete .github/ .gitignore create-release-tarball.sh
+	# if examples exist include them in the tarball
+	if [[ -d "$(EXAMPLESDIR)" ]]; then
+		tar -rf $(TARBALL) --append $(EXAMPLESDIR)
+	fi
+	# include precompiled template pdfs and userguide from manual folder,
+	# the idea being that the userguide can be build from the manual folder
+	# and has everything that it needs to compile. If the release method gets
+	# called this ensures that a precompiled version of the userguide is
+	# included in the tar ball.
+	tar -rf $(TARBALL) --append $(MANUALDIR)
+	# compress
+	gzip $(TARBALL)
+
+release: userguide version clean tarball
+
+#.PHONY: clean
+clean:
+	for dir in $(MODERNCVDIR) $(MANUALDIR); do
+		echo cleaning directory $$dir
+		cd $$dir/
+		rm -fv *.acn *.acr *.alg *.aux *.bcf *.blg *.dvi *.fdb_latexmk *.fls;
+		rm -fv *.glg *.idx *.ilg *.ist *.spl *.lof *.log *.lot *.out *.pdfsync;
+		rm -fv *.run.xml *.snm *.synctex.gz *.tdo *.toc *.vrb *blx.bib *~;
+		if [[ "$$dir" != "." ]]; then
+			cd ..
+		fi
+	done
+
+delete:
+	rm -fv $(TEMPLATEPDF)
+	rm -fv $(MANUALPDF)
+
+deleteexamples:
+	rm -rfv $(EXAMPLESDIR)
+	rm -fv $(MANUALDIR)/*.pdf
```

Comment:
I guess we can merge these simply to `delete`, that should be enough. Also, `delete` should call `clean`.